trillian/migrillian/configpb/config.proto

```diff
@@ -83,6 +83,14 @@ message MigrationConfig {
   // The function that computes LeafIdentityHash for Trillian log entries.
   IdentityFunction identity_function = 12;
 
+  // If set to false (by default), then Migrillian verifies that the tree as
+  // seen by Trillian is consistent with the current STH of the source CT log.
+  // It does so by requesting inclusion proofs corresponding to specific tree
+  // sizes, and computing the expected root hash using them. Doing so requires
+  // the source CT log to expose get-entry-and-proof endpoint (section 4.8 of
```

Comment:
It's not so much that it's incorrect, more that people reading this document might assume they don't need to provide get-entry-and-proof. The RFC is the source of truth for this so I don't think we need some of the detail here.

OK how about this? Replace the text from "Doing so .." with:

"If true then consistency is verified. This verification uses the get-entry-and-proof endpoint (section 4.8 of RFC 6962)."

Reply:
I like this wording, thanks.